### PR TITLE
(DOCSP-7890): added node x509 connection snippet

### PR DESCRIPTION
**Pull request by mongomoe**
Ticket: https://jira.mongodb.org/browse/DOCSP-7890

### CODE REVIEW

File: source/includes/connection-snippets/x509-snippets/node-connection.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  sslCA: fs.readFileSync('/etc/certs/mongodb/ca.pem'),
+  sslKey: fs.readFileSync('/etc/certs/mongodb/client.pem'),
+  sslCert: fs.readFileSync('/etc/certs/mongodb/client.pem')
+});
+
+async function run() {
+  try {
+    await client.connect();
+    const database = client.db("testDB");
+    var collection = database.collection("testCol");
```
Comment (daprahamian) [inline source/includes/connection-snippets/x509-snippets/node-connection.js]: nit: `const collection`

File: source/includes/connection-snippets/x509-snippets/node-connection.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    const docCount = collection.countDocuments({});
```
Comment (daprahamian) [inline source/includes/connection-snippets/x509-snippets/node-connection.js]: need to `await` the `countDocuments`

File: source/includes/connection-snippets/x509-snippets/node-connection.js (hunk 1 above)
Comment (mongomoe) [reply source/includes/connection-snippets/x509-snippets/node-connection.js]: Fixed

File: source/includes/connection-snippets/x509-snippets/node-connection.js (hunk 2 above)
Comment (mongomoe) [reply source/includes/connection-snippets/x509-snippets/node-connection.js]: Fixed

**Comment by mongomoe** [comment]
@daprahamian all comments addressed

**Pull request merged**